infra/modules/kubevela-addons/variables.tf

```diff
@@ -104,3 +104,56 @@ variable "addons_dir" {
   type        = string
   default     = "./addons"
 }
+
+variable "secrets_namespace" {
+  description = "Namespace to deploy secrets to"
+  type        = string
+}
+
+variable "jwt_signature" {
+  description = "K8s secret for Marlowe JWT signature"
+  type        = string
+  sensitive   = true
+}
+
+variable "jwt_signature_input_properties" {
+  description = "Linked with JWT signature secret"
+  type        = string
+  sensitive   = true
+}
+
+variable "gh_oauth_callbackPath" {
+  description = "Linked with gh-oauth secret"
+  type        = string
+  sensitive   = true
+}
+
+variable "gh_oauth_clientID" {
+  description = "Linked with gh-oauth secret"
+  type        = string
+  sensitive   = true
+}
+
+variable "gh_oauth_clientSecret" {
+  description = "Linked with gh-oauth secret"
+  type        = string
+  sensitive   = true
+}
+
+variable "gh_oauth_input_properties" {
+  description = "Linked with gh-oauth secret"
+  type        = string
+  sensitive   = true
+}
+
+variable "iohk_ghcr_creds_dockerconfigjson" {
+  description = "Linked with iohk-ghcr-creds secret"
+  type        = string
+  sensitive   = true
+}
+
+variable "iohk_ghcr_creds_input_properties" {
+  description = "Linked with iohk-ghcr-creds secret"
+  type        = string
+  sensitive   = true
+}
```

Comment:
Wouldn't be better to create just a single variable receiving an object? This will prevent us to adding new variables every time that we need to add another secret.